### PR TITLE
Adjust `field_guid` generation

### PR DESCRIPTION
**Pull request by hackalot805**
Background:
The current logic can create `field_guid` values that may overflow the DB column's length.  Let's adjust it to not do that.

CR:
I changed the prefix of `field_` to `f_` and clipped the use of the `label` to 30 characters.

QA:
I have no idea how to get this qualified properly.  I pulled the code down to my `~/Code/3P/Bootstrap-Form-Builder` folder and ran `make`.  Then I ran `make` in my `Code` folder just to be safe.  Next, I played with adding DC fields to a guide and inspected the DB records.

CC @danielbeardsley 
CC @sterlinghirsh

### CODE REVIEW

**Comment by hackalot805** [comment]
This doesn't look right.  dev_block 💀

**Comment by hackalot805** [comment]
Crap.  Repo is archived.

File: assets/js/models/snippet.js [hunk 1]
```diff
@@ -24,10 +24,10 @@ define([
       if (reduced.field_guid == 'temporary-id') {
          // Make a unique id for the field that will be passed around
          // across edits. Must start with a letter, so we start with
-         // 'field_'.  Includes the label, stepid, date and a random number.
-         // If we get duplicates, I'll eat my hat (figuratively).
-         var newId = 'field_' + reduced.label + '_' + App.stepid + '_' +
-          (new Date().toString()) + '_' + Math.random();
+         // 'f_' for 'field'.  Includes the label (first 30 chars), stepid,
+         // date and a random number.
+         var newId = 'f_' + reduced.label.substr(0, 30) + '_' + App.stepid +
```
Comment (BaseInfinity) [inline assets/js/models/snippet.js:29]: Makes sense as long as the regex is flexible enough on line 31 (I think it is)

**Comment by BaseInfinity** [comment]
CR 👙

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by PanzerKnackers** [comment]
QA 🦚 

Everything here looks to be functioning as intended, all data captures are functional as well as feild_guid being truncated correctly.

<details open><summary>Screenshots</summary>

Old:

image

New:

image

</details>

Saving two data captures with identical names simultaneously.

<details><summary>Screenshots</summary>

image

</details>

A few extra data captures populating correctly

<details><summary>Screenshots</summary>

image

</details>

**Pull request merged**